spec.html

```diff
@@ -38264,16 +38265,16 @@ <h1>JSON.stringify ( _value_ [ , _replacer_ [ , _space_ ] ] )</h1>
       </emu-note>
 
       <emu-clause id="sec-serializejsonproperty" aoid="SerializeJSONProperty">
-        <h1>Runtime Semantics: SerializeJSONProperty ( _key_, _holder_ )</h1>
-        <p>The abstract operation SerializeJSONProperty with arguments _key_, and _holder_ has access to _ReplacerFunction_ from the invocation of the `stringify` method. Its algorithm is as follows:</p>
+        <h1>Runtime Semantics: SerializeJSONProperty ( _state_, _key_, _holder_ )</h1>
+        <p>The abstract operation SerializeJSONProperty with arguments _state_, _key_, and _holder_ performs the following steps:</p>
         <emu-alg>
           1. Let _value_ be ? Get(_holder_, _key_).
```

Comment:
perhaps?
```suggestion
          1. Assert: _state_ is a Record.
          1. Let _value_ be ? Get(_holder_, _key_).
```

Reply:
Hm, is it that helpful since it's not an assert on the specific fields in the record?

Reply:
https://github.com/tc39/ecma262/pull/1890#issuecomment-595904356 suggests it's not